docs/release_notes/2024/dec2024.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 1
+id: December_2024
+description: Release notes for December
+title: December 2024
+tags:
+  - release notes
+  - changelog
+  - december
+---
+
+## 3.4.2.\* (December 13, 2024)
+
+- Prophecy Python libs version: 1.9.27
+- Prophecy Scala libs version: 8.6.0
+
+### Features {#Features342}
+
```

Comment:
we can discuss later too, but what do you think about adding a [small toc](https://docusaurus.io/pt-BR/docs/2.x/markdown-features/toc#inline-table-of-contents) below the feature heading? the h3 headings don't show up in the right toc and maybe someone would want a list of features per release?

Reply:
Added one. LMK what you think.